ksql-common/src/test/java/io/confluent/ksql/util/KsqlConfigTest.java

```diff
@@ -152,6 +162,14 @@ public void shouldCleanStreamsProperties() {
     );
   }
 
+  @Test
+  public void shouldFilterPropertiesForWhichTypeUnknown() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(Collections.singletonMap("you.shall.not.pass", "wizard"));
+    assertThat(
+        ksqlConfig.getAllConfigPropsWithSecretsObfuscated().keySet(),
```

Comment:
If I understand correctly, `ksqlConfig.getKsqlStreamConfigProps` will return the `you.shall.not.pass` config?